support/@trustfractal/data-host/.vscode/launch.json

```diff
@@ -0,0 +1,29 @@
+{
+  "version": "0.2.0",
+  "configurations": [
+    // To debug, make sure a *.spec.ts file is active in the editor, then run a configuration
```

Comment:
Unfortunately, most JSON parsers don't allow comments. So we should remove these.